.editorconfig

```diff
@@ -12,6 +12,9 @@ indent_style = space
 insert_final_newline = true
 charset = utf-8
 
+[*.{sln,*proj}]
+charset = utf-8-bom
```

Comment:
The commit message says "VS needs projects to use byte order marks". I tried to google for this, but found answers in both directions.
Is the reason the same as in DotNetAnalyzers/StyleCopAnalyzers#1142? (since the first comment in https://marketplace.visualstudio.com/items?itemName=SergeyVlasov.FixFileEncoding#review-details seems to indicate that some people want sln, csproj ... files without BOM).

Reply:
Ah. If you ever edit the solution or project file, VS forces a resave with BOM and so git shows an edit on the top row back and forth for no good reason.

Reply:
Fwiw I strongly disagree with byte order marks. But so long as tooling forces thrashing like this on us, I'd rather not ask everyone to keep an eye on what VS does and squash it out of their PRs.

Reply:
Ok. I read "VS **needs** projects to use byte order marks" as this change was required for VS to work, but I'm all for reducing noise in reviews.

Reply:
That wasn't clear wording, I'm sorry!